incubator/kube-downscaler/values.yaml

```diff
@@ -0,0 +1,47 @@
+# Default values for kube-downscaler.
+# Declare variables to be passed into your templates.
+
+# How many kube-downscaler pods should run in deployment
+replicaCount: 1
+name: kube-downscaler
+debug:
+  enable: 'False'
+
+# Namespace where kube-downscaler is going to be deployed.
+namespace:
+  # Deployment will query all namespaces if left empty:
+  active_in: ''
+
+# Default is 1 minute.
+interval: 60
+
+rbac:
+  # If true, create & use RBAC resources
+  create: false
```

Comment:
please follow https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md#rbac-resources-should-be-created-by-default

should be true by default

Reply:
Done, if you see anything else do not hesitate 😄